hmda/src/test/scala/hmda/validation/rules/lar/quality/_2022/Q659_2Spec.scala

```diff
@@ -0,0 +1,28 @@
+package hmda.validation.rules.lar.quality._2022
+
+import hmda.model.filing.lar.LarGenerators._
+import hmda.model.filing.lar.LoanApplicationRegister
+import hmda.validation.rules.EditCheck
+import hmda.validation.rules.lar.LarEditCheckSpec
+
+class Q659_2Spec extends LarEditCheckSpec {
+  override def check: EditCheck[LoanApplicationRegister] = Q659_2
+
+  val invalidExemptCodes = List("999", "99999")
+  property("Exempt field should not have a value of 999, or 99999") {
+    forAll(larGen) { lar =>
+      invalidExemptCodes.foreach(invalidExemptCode => {
+        lar.copy(applicant = lar.applicant.copy(age = invalidExemptCode.toInt)).mustFail
```

Comment:
@lchen-2101 this should be sprinkled with scenarios in which is passes the validation check